dev/PushNotifications/PushNotificationRegistrationToken.h

```diff
@@ -0,0 +1,25 @@
+﻿#pragma once
+#include "Microsoft.Windows.PushNotifications.PushNotificationRegistrationToken.g.h"
+
+namespace winrt::Microsoft::Windows::PushNotifications::implementation
+{
+    struct PushNotificationRegistrationToken : PushNotificationRegistrationTokenT<PushNotificationRegistrationToken>
+    {
+        PushNotificationRegistrationToken() = default;
```

Comment:
Delete this. You don't want to let people create a `PushNotificationRegistrationToken` with a garbage cookie.